app/controllers/admin/assets_controller.rb

```diff
@@ -33,6 +33,11 @@ def mark_group_as_spam
       redirect_back(fallback_location: root_path)
     end
 
+    def delete
+      @asset.soft_delete
```

Comment:
Couple things here

1) We'll have to handle restoring 
2) We'll have to handle soft deleting and restoring relations (comments, listens)
3) We don't want 2 concepts of asset deletion in the wild. We probably want to make sure the public "delete" behavior which moderators and users can do from AssetsController#delete aligns with this behavior and also soft deletes w/ relations.